[Instrumentation.Elasticsearch] Update readme - new library (#866)
open-telemetry/opentelemetry-dotnet-contrib: src/OpenTelemetry.Instrumentation.ElasticsearchClient/README.md

[![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.ElasticsearchClient.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.ElasticsearchClient)
[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.ElasticsearchClient.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.ElasticsearchClient)

Automatically instruments events emitted by the [NEST/Elasticsearch.Net](https://www.nuget.org/packages/NEST)
client library.

> Newer version [Elastic.Clients.Elasticsearch](https://www.nuget.org/packages/Elastic.Clients.Elasticsearch)

Author: Any version of Elastic.Clients.Elasticsearch. As I understand, there is a goal to fully drop NEST.

> brings native support for OpenTelemetry. To instrument this version you need

Reviewer: ```suggestion
brings native support for OpenTelemetry. To instrument it you need
```

});
```

> ## Elastic.Clients.Elasticsearch

Reviewer: Personally, I would move this section before "NEST/Elasticsearch.Net". Because "Elastic.Clients.Elasticsearch" is newer and this is what the users should use 😉

Author: I would keep it as is it is. Readme is strictly related to the package.
I would like to have just a note that it will not work with newer versions.

Reviewer: I would like to resolve this comment, but I cannot 😉

> Newer version [Elastic.Clients.Elasticsearch](https://www.nuget.org/packages/Elastic.Clients.Elasticsearch)

Reviewer: ```suggestion
[Elastic.Clients.Elasticsearch](https://www.nuget.org/packages/Elastic.Clients.Elasticsearch),
that deprecates `NEST/Elasticsearch.Net`,
```
